Modules/ip.py

```diff
@@ -0,0 +1,42 @@
+import main
+import os
+from requests import get
+import time
+
+def Init(module: main.Module):
+    module.color = '1010101'
+
+def Update(module: main.Module):
+    # Check if not a timer for the module is set
+    ip_path = "module_data/ip/timer.txt"
```

Comment:
Use a local variable instead of a file for timers

Reply:
As far as I understand it, every cycle of the swaybar this file is run anew. So if i were to use a local variable, it would get dumped each time.